spec/tests/posts/posts_show_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+# rubocop:disable Metrics/BlockLength
```

Comment:
- [OPTIONAL] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice.Instead of disabling `rubocop`, it will be nice to fix the issue.